docs/source/specs/openapi.json

```diff
@@ -984,7 +998,7 @@
             "content": {
               "application/json": {
                 "schema": {
-                  "$ref": "#/components/schemas/RolePagination"
+                  "$ref": "#/components/schemas/RolePaginationV2"
```

Comment:
Can we maybe name this in line with the other change, like `RolePaginationDynamic`?

Reply:
Sure, fixed